docs/un-supported-patch-properties.md

```diff
@@ -14,7 +14,7 @@ ARM OpenAPI(swagger) specs
 
 ## Output Message
 
-The patch operation body parameter schema should not contains property {propertyName}.
+The patch operation body parameter schema should not contain top level "id", "name", "type", "location" as writable properties. This is because these properties are expected to be either readOnly or immutable.
```

Comment:
```suggestion
The patch operation body parameter schema should not contain top level "id", "name", "type", or "location" as writable properties. This is because these properties are expected to be either readOnly or immutable.
```

Reply:
hmm including them as top-level would not require them to be writable.

Reply:
The properties should either not exist in the patch request body or if they are specified, they must be marked as readOnly or immutable